lua/entities/gmod_wire_expression2/core/number.lua

```diff
@@ -408,6 +431,10 @@ __e2setcost(2) -- approximation
 local deg2rad = pi / 180
 local rad2deg = 180 / pi
 
+registerFunction("huge", "", "n", function(self, args)
```

Comment:
Well I mean, we have the constant pi and a function pi. I could see this being convenient in some cases.